src/main/java/gov/cabinetoffice/gapfindapiadmin/controllers/ApiKeyController.java

```diff
@@ -33,28 +34,34 @@ public class ApiKeyController {
     public static final String REVOKE_API_KEY_CONFIRMATION_PAGE = "revoke-api-key-confirmation";
     public static final String ERROR_PAGE = "error-page";
     public static final String SUPER_ADMIN_PAGE = "super-admin-api-keys";
-    public static final String SUPER_ADMIN_ROLE = "SUPER_ADMIN";
 
     private final ApiKeyService apiKeyService;
     private final ApiGatewayService apiGatewayService;
     private final PaginationHelper paginationHelper;
+    private final UserServiceConfig userServiceConfig;
 
     @GetMapping
     @PreAuthorize("hasAuthority('TECHNICAL_SUPPORT')")
     public ModelAndView showKeys() {
         final GrantAdmin grantAdmin = (GrantAdmin) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
         final String departmentName = grantAdmin.getFunder().getName();
 
-        return new ModelAndView(ORGANISATION_API_KEYS_PAGE)
+        ModelAndView model = new ModelAndView(ORGANISATION_API_KEYS_PAGE)
                 .addObject("apiKeys", apiKeyService.getApiKeysForFundingOrganisation(grantAdmin.getFunder().getId()))
-                .addObject("departmentName", departmentName);
+                .addObject("departmentName", departmentName)
+                .addObject("signOutUrl", userServiceConfig.getLogoutUrl());
+
+        return apiKeyService.isAdmin() ? model.addObject("navBar", apiKeyService.generateNavBarDto()) : model;
```

Comment:
This is personal style but I'm not sure I would leave this logic until a ternary at the very end. I'd probably just do an `isdmin` check and conditionally add to the model then return the model separately. 

Can't really explain why but we can discuss tomorrow. Same applies for most of these methods, especially the ones where there is a number of checks inside the ternary such as `isAdmin || isSuperAdmin`